gradle-plugin/src/main/groovy/io/realm/gradle/Realm.groovy

```diff
@@ -29,12 +29,14 @@ class Realm implements Plugin<Project> {
     @Override
     void apply(Project project) {
         // Make sure the project is either an Android application or library
-        def isAndroidApp = project.plugins.withType(AppPlugin)
-        def isAndroidLib = project.plugins.withType(LibraryPlugin)
-        if (!isAndroidApp && !isAndroidLib) {
+        def appPlugins = project.plugins.withType(AppPlugin)
```

Comment:
No i think? otherwise below would be really really really strange:
```
def androidPlugin = isAndroidApp ? isAndroidApp.first() : isAndroidLib.first()
```